File: 1013.js
let input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8') 
let lines = input.split('\n')

const pegarValores = (line) => line.split(" ").map(a => +a);  
let [a , b, c] = pegarValores(lines.shift());

let result = ((a + b + Math.abs(a - b)) / 2)
let maior = ((result + c + Math.abs(result - c)) / 2)
console.log(`${maior} eh o maior`)

File: 1013.py
# 1013 - @crisfeldk

entrada = input().split(" ")
a, b, c = entrada
omaior = (int(a) + int(b) + abs (int(a) - int(b))) / 2
restante = (int(omaior) + int(c) + abs (int(omaior) - int(c))) / 2
print("%d eh o maior"%restante)
